client/e2e/poms/profile.ts

```diff
@@ -43,7 +65,7 @@ export class ProfilePOM {
         if (labelText === "Phone Number*") {
           await this.page.getByLabel(labelText).fill("987 654 3210"); //Format should be ### ### ####
         } else {
-          await inputField.fill(`E2E TEST ${labelText}`);
+          await inputField.fill(`E2E ${labelText}`);
```

Comment:
I'd suggest naming this function `fillRequiredFields` (or something) because as far as I can tell, it doesn't check for successful form submission. (`updateFail` is good as-is I think because it counts validation errors)

Reply:
Change return to `await expect(isSuccessExisted).toBe(true);`